k8s/eks/README.md

```diff
@@ -0,0 +1,284 @@
+sudo mv /tmp/eksctl /usr/local/bin
+eksctl version
+```
+
+
+## Step by step installation guide
+
+1. Pull the infra repo from here:
+
+git clone https://github.com/testground/infra.git
+
+and the testground repo from here:
+
+git clone https://github.com/testground/testground
+
+2. You need to populate the `.env` file with the needed parameters before creating the cluster. It is located in `infra/k8s/eks/`
+
+3. Run the installation script `./testground_install.sh`; note that it requires the `.env` file to be populated in order to run properly. In case of any missing variables, the setup will fail and you will get an error message
+
+The script explained in short:
+
+- Takes approx. 15-20 minutes to provision a cluster with 2 worker nodegroups, along with all other necessary resources and cluster workloads
+- All installation steps have been divided into functions, located in the `eks/bash/functions.sh` script
+- The script calls functions one by one
+- Output of every step is logged into a master log file that can be retrieved and reviewed, or even tailed in real time
+- It can also be used to uninstall a provisioned cluster; one of the first steps in the script is checking for existing resources
+- The `functions.sh` file relies on the content of `eks/yaml` - all cluster resources are located inside this folder
+- If you need to scale the worker nodegroups, or introduce any other changes, you are able to edit the `functions.sh` script
+- Once the cluster is created, eksctl will automatically switch the context to the new cluster. If you create another cluster, eksctl will once again switch the context to the newest cluster. If you wish to switch to another cluster, issue:
+
+```
+kubectl config get-contexts
+
+# currently selected cluster will be mark with an asterisk (*)
+# then switch contexts to the desired cluster
+
+kubectl config use-context $USERNAME@$CLUSTER_NAME.$AWS_REGION.eksctl.io
+```
+
+4. Create the `.env.toml` file inside the `testground` folder and populate it (explained here as well https://github.com/testground/docs/blob/master/getting-started.md#configuration-envtoml):
+
+**Note: The endpoint refers to the `testground-daemon` service, so depending on your setup, this could be, for example, a Load Balancer fronting the kubernetes cluster and forwarding proper requests to the `tg-daemon` service, or a simple port forward to your local workstation.**
+
+`.env.toml` example:
+
+```
+["aws"]
+region = "$YOUR_REGION" # where the cluster will be created
+[client] # Note: Only one endpoint can be active, please refer to steps 5. and 6. for clarification
+endpoint = "http://localhost:8080" # if using port-forwarding
+endpoint = "http://acca15c6ah3eh45n68ad6052c20ba9ec-1569872837.eu-west-3.elb.amazonaws.com:80" # if using LB (default)
+user = "myname"
+```
+
+5. Using the simple port-forward solution
```

Comment:
I would leave it there for the sake of user preference, and having seen how useful it can be as it is fairly simple to use.
And I believe that people could benefit from that information.

Reply:
As in https://github.com/testground/infra/pull/78#discussion_r985626524, I'd suggest describing port-forwarding setup separately then. What I'd like us to achieve is a simple README that takes you through the most common path.

Reply:
As answered [here](https://github.com/testground/infra/pull/78#discussion_r985634903), we could move the port forward to a different section.

Reply:
Done, answered [here](https://github.com/testground/infra/pull/78#discussion_r989377865)